src/lib_ccx/ccx_encoders_common.c

```diff
@@ -1045,203 +1041,204 @@ int encode_sub(struct encoder_ctx *context, struct cc_subtitle *sub)
 		// Write to a buffer that is later s+plit to generate split
 		// in sentences
 		if (sub->type == CC_BITMAP)
-			wrote_something = write_cc_bitmap_to_sentence_buffer(sub, context);
+			sub = reformat_cc_bitmap_through_sentence_buffer(sub, context);
```

Comment:
This is the most important point in this PR.

SBS works as **transformation filter**. It takes incoming subs, and convert them to other subs with `sub->type = CC_TEXT`. This transformation occurred before all other encoders, so other encoders remain unchanged.